src/main/java/org/example/Model/Account.java

```diff
@@ -1,4 +1,87 @@
 package org.example.Model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+import java.util.UUID;
+
+@Entity
+@Table(name = "account")
 public class Account {
+
+    @Id
+    private UUID account_id;

```

Comment:
Redundant naming; 99% of the time just keeping it as `id` is sufficient:
```suggestion
    private UUID id;
```